### PR TITLE
(fix) Resolving Duplicate completed form display issue in configurable form view

### PR DESCRIPTION
**Pull request by donaldkibet**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [x] My work includes tests or is validated by existing tests.

## Summary
This resolves an existing problem where configurable forms are duplicated when switching between 'completed' and 'available' categories. Additionally, it includes enhancements to the mutate function. Now, once a form has been successfully completed, the function will automatically update and move it from the 'available' to 'completed' section, ensuring our list remains accurate and up-to-date.

### CODE REVIEW

File: packages/esm-patient-forms-app/src/forms/forms.component.tsx [hunk 1]
```diff
@@ -82,6 +82,7 @@ const Forms: React.FC<FormsProps> = ({ patientUuid, patient, pageSize, pageUrl,
         pageUrl={pageUrl}
         urlLabel={urlLabel}
         error={error}
+        mutateForms={mutateForms}
```
Comment (ibacher) [inline packages/esm-patient-forms-app/src/forms/forms.component.tsx:85]: Might be better to make this part of a React Context so we don't have to keep prop-drilling.

Comment (donaldkibet) [reply packages/esm-patient-forms-app/src/forms/forms.component.tsx:85]: Sure i will fix this in a subsequent PR

**Comment by vasharma05** [comment]
Hi @donaldkibet !
Any update on this PR?
Thanks!

**Comment by donaldkibet** [comment]
We don't require this anymore

**Pull request closed without merging**